coalib/settings/ConfigurationGathering.py

```diff
@@ -385,6 +386,21 @@ def gather_configuration(acquire_settings,
     local_bears, global_bears = fill_settings(sections,
                                               acquire_settings,
                                               log_printer)
+
+    arg_parser = default_arg_parser() if arg_parser is None else arg_parser
+
+    for arg_key, arg_value in sorted(
+            vars(arg_parser.parse_args(arg_list)).items()):
+        if arg_key == 'exclude_sections':
+            if arg_value is not None:
```

Comment:
This check is not needed because if `arg_value` is None, the `for` iteration below wouldn't run anyway.